google/auth/aws.py

```diff
@@ -106,9 +106,9 @@ def get_request_options(
             Mapping[str, str]: The AWS signed request dictionary object.
         """
         # Get AWS credentials.
-        access_key = aws_security_credentials.get("access_key_id")
-        secret_key = aws_security_credentials.get("secret_access_key")
-        security_token = aws_security_credentials.get("security_token")
+        access_key = aws_security_credentials.accessKeyId
+        secret_key = aws_security_credentials.secretAccessKey
+        security_token = aws_security_credentials.sessionToken
```

Comment:
nit: Simplify the code by using the data class directly, instead of unrolling it into variables.